Source/WebKit/UIProcess/_WKWebViewPrintFormatter.mm

```diff
@@ -71,6 +73,20 @@ - (WKWebView *)_webView
     return static_cast<WKWebView *>(view);
 }
 
+- (BOOL)_shouldUseResponsivePrintRendering
```

Comment:
Let's keep the confusing "responsive" name as confined as possible (not here, for example)